baus/subsidies.py

```diff
@@ -248,27 +181,17 @@ def inclusionary_housing_revenue_reduction(feasibility, units):
     households = orca.get_table("households")
     buildings = orca.get_table("buildings")
     parcels_geography = orca.get_table("parcels_geography")
-    policy = orca.get_injectable("policy")
 
-    if orca.get_injectable("scenario") in policy["inclusionary_d_b_enable"]:
-        h = orca.merge_tables("households",
-                              [households, buildings, parcels_geography],
-                              columns=["juris_name",
-                                       "income",
-                                       "pba50chcat"])
-        AMI = h.groupby(h.pba50chcat).income.quantile(.5)
-    elif orca.get_injectable("scenario") in policy["inclusionary_fb_enable"]:
-        h = orca.merge_tables("households",
-                              [households, buildings, parcels_geography],
-                              columns=["juris_name",
-                                       "income",
-                                       "fbpchcat"])
-        AMI = h.groupby(h.fbpchcat).income.quantile(.5)
-    else:
-        h = orca.merge_tables("households",
-                              [households, buildings, parcels_geography],
-                              columns=["juris_name", "income"])
-        AMI = h.groupby(h.juris_name).income.quantile(.5)
+    run_setup = orca.get_injectable("run_setup")
+    policy = orca.get_injectable("policy")
+    # determine the geography type to use by reading the "type" that the first inclusionary rate is applied to, 
+    # since we tend to use the same geography type for applying all of the inclusionary rates
+    if run_setup["run_inclusionary_strategy"]:
```

Comment:
Yes it's expected, because if the strategy is not run it still needs to apply the inclusionary policies that are on the ground today (the `default`). 

I'm separating out existing policy/future strategies/k-factor etc. inputs via PR #3. That would include breaking inclusionary policy into `strategies.yaml` and `existing_policy.yaml` so one could locate those more easily. I would also ideally separate the code out so that there is a set of strategy code/scripts/orca steps that run when strategies are enabled. Currently that's slated under a future PR, though.

Reply:
Got it. Sounds good with the segmentation in PR 3 - that will make this sort of thing clearer.